src/components/containers/notification/Notification.js

```diff
@@ -146,6 +146,13 @@ function renderCallsToAction(callsToAction) {
       {callsToAction.map((callToAction, index) => {
         const buttonStyleType = index === 0 ? 'primary' : 'default';
 
+        if (React.isValidElement(callToAction)) {
```

Comment:
I do not know of a reliable way to define a "Button only" check when the JSX could potentially be a component that returns a Button, which is a valid scenario. I do share the concern, but I don't know how to enforce that restriction given the contents of the element object we would get back when a stateful button is provided.